odoo/addons/base/views/onboarding_views.xml

```diff
@@ -24,110 +24,68 @@
                             <t t-if="text_completed" t-esc="text_completed" />
                             <t t-else="">Nice work! Your configuration is done.</t>
                         </span>
-                            <a type="action" class="btn btn-primary" data-toggle="collapse" href=".o_onboarding_container"
-                            t-att-data-model="close_model" t-att-data-method="close_method">
-                                Close
-                            </a>
+                        <a type="action" class="btn btn-primary ms-4" data-toggle="collapse" href=".o_onboarding_container" t-att-data-model="close_model" t-att-data-method="close_method">
+                            Close
+                        </a>
                     </div>
                 </div>
             </div>
         </div>
     </template>
+
     <template id="onboarding_step">
-        <div t-att-class="'o_onboarding_step ' + ('o_onboarding_done' if state in ('done', 'just_done') else 'o_onboarding_current') + (' o_onboarding_just_done' if state == 'just_done' else '')">
-            <t t-call="base.onboarding_confetti" />
-            <div class="o_onboarding_progress d-inline-block progress">
-                <div class="progress-bar"></div>
+        <div class="o_onboarding_step position-relative d-flex flex-column align-items-center justify-content-start text-center" t-att-data-step-state="state">
+            <img t-if="state == 'just_done'" class="o_onboarding_confetti position-absolute w-100" src="/base/static/img/onboarding_confetti.svg" alt="o_onboarding_confetti"/>
+
+            <div class="o_onboarding_step_side d-flex pt-0 flex-grow-1">
+                <div class="o_onboarding_progress position-absolute"/>
+                <span t-attf-class="o_onboarding_dot fa fa-check d-inline-block rounded-circle small {{state != 'not_done' and 'text-white' or ''}}"/>
             </div>
-            <span class="o_onboarding_dot d-inline-block fa fa-check" />
-            <div class="o_onboarding_step_content">
-                <div class="o_onboarding_step_content_info">
-                    <h4 class="o_onboarding_step_title mt16">
-                        <a type="action" data-reload-on-close="true" role="button"
-                            t-att-data-method="method" t-att-data-model="model">
-                            <t t-esc="title" />
-                        </a>
-                    </h4>
-                    <p class="mb16">
-                        <t t-esc="description" />
-                    </p>
+
+            <div class="o_onboarding_step_content py-3 flex-grow-1 d-flex flex-column align-items-center justify-content-around">
+                <div class="o_onboarding_step_content_info mb-3">
+                    <a type="action" data-reload-on-close="true" role="button" t-att-data-method="method" t-att-data-model="model">
+                        <h4 class="o_onboarding_step_title text-white" t-esc="title"/>
```

Comment:
As we change the template anyway, shouldn't we use this opportunity to migrate form `t-esc/t-raw` to `t-out` ?